### DISCUSSION TITLE
Need more ressources or architectures of how Zephyr support MAX3421E

### DISCUSSION THREAD

**Discussion by S0UL4** [zephyrproject-rtos/zephyr#70788] [Q&A]
Hello everyone,

Has anyone here successfully used the MAX3421E with a Nordic board to achieve bidirectional communication with a USB device, specifically for both reading and writing data ?

Thank u guys in advance!

**Comment by ndrs-pst** [comment]
@S0UL4, I'm not sure if you've already tried the USB Shell sample. If not, please take a look here: https://docs.zephyrproject.org/3.6.0/samples/subsys/usb/shell/README.html."
